packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -190,7 +190,16 @@ Future<void> _runBuild(
 
   // MSBuild sends all output to stdout, including build errors. This surfaces
   // known error patterns.
-  final RegExp errorMatcher = RegExp(r':\s*(?:warning|(?:fatal )?error).*?:');
+  final RegExp errorMatcher = RegExp(
+	<String>[
```

Comment:
Please use spaces:

```suggestion
    <String>[
```

Reply:
as an aside, we should add a test for this in analyze.dart.

Reply:
please still fix this

Reply:
Fixed